index.html

```diff
@@ -224,11 +221,11 @@ <h2 class="Tonic">Multi-Post <br>Stories</h2>
             </div>
         </div>
     </section>
-    <div id="none" class="bgc">
+    <div class="bgc">
         <section class="about">
-            <div class="aboutmyself" id="about-me">
+            <div class="aboutmyself">
                 <h2 class="heading">About<br> Myself</h2>
-                <article id="mainss" class="main">
+                <article class="main">
                     <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 </article>
                 <h2 class="connect">LET'S CONNECT</h2>
```

Comment:
- [ ] First, you will notice the content on the right is wider, yes it is 60% width and the left one is 40%, kindly adapt it to your project.

### Figma
![image](https://github.com/AbednegoKibitok/module1microverse/assets/86200356/577a63cd-1d13-45bf-87d6-a484e9ddbfae)

### Your project
![image](https://github.com/AbednegoKibitok/module1microverse/assets/86200356/1e0a6009-5211-4a33-9dfe-046ad81774e8)

- [ ] Secondly, you will notice the font of the text `About myself` is quite not tick as Figma's own, I think you missed the font-weight. Kindly fix it. Take note that that same text is on one line in the Figma design, but yours broke to the second line.